datalad/core/distributed/push.py

```diff
@@ -266,6 +265,23 @@ def __call__(
             for i, ds in pbars.items():
                 log_progress(lgr.info, i, 'Finished push of %s', ds)
         if not matched_anything:
+            potential_remote = False
+            if not to and path:
+                # if we get a remote name without --to, provide a hint
+                sr = ds_repo.get_remotes(**get_remote_kwargs)
+                potential_remote = [
+                    p for p in ensure_list(path) if p in sr
+                ]
+            if potential_remote:
+                msg = "It seems like you specified a sibling name {} " \
+                      "as a path. Forgot to use --to?".format(potential_remote)
```

Comment:
I am unsure about the communication style. I would personally do:

```
'{} matches a sibling name and not a path. Forgot --to ?'.format(potential_remote)
```

Just statement of a fact, plus suggestion. Less (unnecessary) speculation on intent. WDYT?

Reply:
I like this suggestion. Thanks!